build.sbt

```diff
@@ -1813,6 +1810,7 @@ lazy val experimentalFlinkTableApiComponents = (project in flink("components/dev
         "org.apache.flink" % "flink-table-api-java-bridge" % flinkV,
         "org.apache.flink" % "flink-table-planner-loader"  % flinkV,
         "org.apache.flink" % "flink-table-runtime"         % flinkV,
+        "org.apache.flink" % "flink-clients"               % flinkV,
```

Comment:
This is added since the `flinkTable.jar` is loaded on classpath of minicluster and it needs flink-clients. Ultimately I think we should have a better approach for loading things like minicluster dependencies / formats / connectors, but I don't see any simple solution.